Gemfile

```diff
@@ -68,6 +68,7 @@ gem 'http_accept_language', require: false
 # behavior, we will push these versions up when upgrading ember
 gem 'ember-rails', '0.18.5'
 gem 'discourse-ember-source', '~> 3.12.2'
+gem 'discourse-fonts', git: 'https://github.com/discourse/discourse-fonts.git'
```

Comment:
Can you publish that gem instead of using git directly?

Reply:
Sure, I can do that.